bridges/zap/zap_encoder.go

```diff
@@ -0,0 +1,211 @@
+package zap
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"time"
+
+	"go.opentelemetry.io/otel/log"
+	"go.uber.org/zap/zapcore"
+)
+
+// not optimized yet
+// this file implements object and array encoder - similar to memory encoder by zapcore
```

Comment:
You should remove development comment here.

If you based this code on another project you need to provide a link. If it was copied in any way it's license needs to be reference in this file as well.